src/amo/css/inc/mixins.scss

```diff
@@ -17,7 +17,14 @@
 @mixin font-family-inter() {
   font-family: 'Inter', sans-serif;
   @supports (font-variation-settings: normal) {
-    font-family: 'Inter Var', sans-serif;
+
```

Comment:
`stylelint` says there should be an empty line before the comment. This is checked by a hook at commit time. However, running `prettier`, which the CI `check` does, expected that line to be removed...